src/ToggleSwitch/ToggleSwitch.tsx

```diff
@@ -187,7 +195,7 @@ const ToggleKnob = styled.div<{checked?: boolean; disabled?: boolean}>`
   }
 
   ${props => {
-    if (props.disabled) {
+    if (props['aria-disabled']) {
       return css`
         border-color: ${get('colors.switchTrack.disabledBg')};
       `
```

Comment:
```suggestion
      return css`
        @media (forced-colors: active) {
          color: GrayText;
        }
        
        border-color: ${get('colors.switchTrack.disabledBg')};
      `
```

Adds the disabled state to be visually present in high contrast mode.